### DISCUSSION TITLE
Evolving the governance of the Open Practice Library

### DISCUSSION THREAD

**Discussion by kattekrab** [openpracticelibrary/openpracticelibrary#2064] [Announcements]
We're evolving, and formalising the governance of the Open Practice Library. 

Currently, the OPL has a a single product owner, and a somewhat loose, informal, advisory community collective.  There's been some "generally accepted agreements" about how things work, and a little bit of documentation to support that.  

But largely, it's fairly structureless.

As we grow, we're going to need something a little bit more formal, and lot more transparent! But, in the spirit of our general approach to these sorts of things, we want "just enough" structure, and no more. We can always evolve again, as our needs dictate.

So, with that, I'd like to invite, and warmly welcome our community to contribute ideas, suggestions, cautions, and concerns as we work to evolve the way we continue to support, and contribute to the Open Practice Library.

Steps I've taken so far:
- Proposed (informally) that we set up a product council, rather than having just one product owner.
- Set OPL up with Open Collective so we can make the financial side transparent, and easier for more people to contribute small amounts of treasure.

What do you think?
What do you suggest?
What are you curious about?

**Comment by eliseelk** [comment]
I think this sounds like a great idea.
How would you imagine the council to work? How many people? Do different geographic locations need to be considered?

I think Open Collective will be very beneficial to OPL and for those who want to contribute financially.

I would like to open up the conversation and decision making to more people, but I am not sure of the best way to do this.

**Comment by mtakane** [comment]
I'm in favor of adoption something that includes soliciting input from various platforms beyond just Github when it comes to direction of the product. And am supportive of having a bit more of a centralized way to communicate information/decisions. 

I fear creating a biased echo chamber if we were to make a decision hearing only from a subset group of personas of the OPL. Having a council that meets somewhat regularly could be helpful in overcoming this.

**Comment by Oleghgh** [comment]
I mean there too
